### DISCUSSION TITLE
Editorial Policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#35] [Added]
### Type name

Editorial Policy

### Alternative type names

Editorial Guidelines

### Writer

service provider

### Audience

end user

### Object

writing and publishing standards and principles

### Examples

https://www.bbc.com/editorialguidelines/guidelines - Editorial Guidelines - BBC
https://misinforeview.hks.harvard.edu/editorial-policies/ - Editorial Policies - HKS Misinformation Review
https://www.bleepingcomputer.com/ethics-statement/ - ETHICS STATEMENT - BleepingComputer
https://lawandcrime.com/ethics-diversity-policy/ - Corrections, Ethics & Diversity Policy - Law&Crime
https://themarkup.org/ethics - Ethics Policy - The Markup
https://bioresources.cnr.ncsu.edu/about-the-journal/editorial-policies/ - Editorial Policies - BioResources
https://www.washingtonpost.com/policies-and-standards/ - Policies and Standards - Washington Post

**Comment by MattiSG** [comment]
## Alternative type names

- I don't think the plural (“Editorial Policies”) is relevant, as this could be applied everywhere.

## Object

Since the triptych is to be read as a “commitment taken on [object]”, it shouldn't start with a verb: “define writing and publishing standards, guidelines, and principles” → “writing and publishing standards and principles”

**Comment by MattiSG** [comment]
I believe that #42 should be merged into this one, see https://github.com/OpenTermsArchive/terms-types/discussions/42#discussioncomment-10198625

**Comment by clementbiron** [comment]
This terms type suggestion has been accepted and will soon be added.
